content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/clientlibs/ngdmsmartcrop/js/smartcropaction.js

```diff
@@ -18,11 +18,15 @@
     "use strict";
     ns.image.v3.actions.smartCrop = function() {
         const editable = this;
-        authorNs.DialogFrame.openDialog(new ns.image.v3.smartCropDialog(editable));
+        if (this.authorNs) {
```

Comment:
why not `if (authorNs)` ?

Reply:
It does not work without `this` - some error appears in the console even before this code is hit:
<img width="903" alt="authorNs" src="https://github.com/adobe/aem-core-wcm-components/assets/20440690/0dc42a9b-053f-47d4-9256-f2e44c9b6300">